src/status_im2/contexts/activity_center/notification/reply/view.cljs

```diff
@@ -37,10 +38,8 @@
     nil))
 
 (defn view
-  [{:keys [author chat-name chat-id message read timestamp]}]
-  (let [chat                    (rf/sub [:chats/chat chat-id])
-        community-id            (:community-id chat)
-        is-chat-from-community? (not (nil? community-id))
+  [{:keys [author chat-name community-id chat-id message read timestamp]}]
+  (let [is-chat-from-community? (not (string/blank? community-id))
```

Comment:
Could you just rename this binding without the prefix `is-`?

Reply:
Updated 👍